pages/how/auth.mdx

```diff
@@ -82,11 +82,13 @@ const mutators = {
       {nodeID, content}: {nodeID: string, content: string}) {
     const node = await getNode(tx, nodeID);
     const auth = tx.auth as MyAuthData;
-    if (auth === undefined || // auth is always undefined on client
```

Comment:
What was wrong with the prev, was it a compile error or something? I think the documentation should still show some form of checking the presence of `auth`, not `window`.